### ISSUE TITLE
Logging: log file and line for LLVM IR instruction

### ISSUE THREAD

**Issue by PLukas2018** [viktormalik/diffkemp#289]
When using `diffkemp` with `-d` option it logs compared instructions. I think it could be useful for debugging to log also the file and line of C source file which the instructions belongs to. This information should be able to get from dbg info of LLVM IR.

**Comment by lenticularis39** [comment]
Good idea, that shouldn't be too hard to implement and would improve the debugging experience.
